daemons/gptp/common/ptp_message.cpp

```diff
@@ -933,7 +937,7 @@ void PTPMessageFollowUp::processMessage(IEEE1588Port * port)
 	sync_arrival = sync->getTimestamp();
 
 	delay = port->getLinkDelay();
-	if ((delay = port->getLinkDelay()) == 3600000000000) {
+	if ((delay = port->getLinkDelay()) == 3600000000000) { /*TODO: ONE_WAY_DELAY_DEFAULT*/
```

Comment:
There is a magic number problem here (of my creation). This is not meant to be a "default" value it is supposed to be an incorrect value that should not be used (aborting processing).  Maybe the code should be something like:

if(( delay = port->getLinkDelay()) == INVALID_LINKDELAY ) {